src/Module/Profile.php

```diff
@@ -63,14 +63,7 @@ public static function rawContent()
 				System::jsonExit($data, 'application/activity+json');
 			} elseif (DBA::exists('userd', ['username' => self::$which])) {
 				// Known deleted user
-				$data = [
-					'@context' => ActivityPub::CONTEXT,
-					'id' => self::getApp()->getBaseUrl() . '/profile/' . self::$which,
-					'type' => 'Tombstone',
-					'published' => DateTimeFormat::utcNow(DateTimeFormat::ATOM),
-					'updated' => DateTimeFormat::utcNow(DateTimeFormat::ATOM),
-					'deleted' => DateTimeFormat::utcNow(DateTimeFormat::ATOM),
-				];
+				$data = ActivityPub\Transmitter::getDeletedUser(self::$which);

```

Comment:
Wouldn't ```getTombstoneForUsername``` a better name? "Tombstone" because the type is "tombstone" and this "ForUsername" because we possibly could have similar stuff for user ids or for item ids or other objects.

Reply:
Possibly, I took after the name `getProfile` but it can be changed. `createNote` actually can return a `type: "Tombstone"` as well but it isn't explicit in the function name.

Reply:
Ah okay.